tests/test_branch.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+from contextlib import contextmanager
+from datetime import datetime, timedelta
+import os
+from pathlib import Path
+from tempfile import TemporaryDirectory
+import subprocess
+import unittest.mock
+from typing import Any, BinaryIO, Iterator
+from nix_update.version import fetch_latest_version
+from nix_update.version.version import VersionPreference
+import conftest
```

Comment:
Your editor should have some sort of python imports sort. That should be run on this.